src/components/HomeGroupItem.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <b-list-group-item
+    :to="{ path: 'recordings', query: query}"
+    class="d-flex justify-content-between align-items-center"
+  >
+    {{ group.groupname }}
+    <b-badge
+      v-if="count > 0"
+      pill
+      variant="primary"
+      class="ml-auto">
+      {{ count }}
+    </b-badge>
+  </b-list-group-item>
+</template>
+
+<script>
+
+import {mapState} from 'vuex';
+import api from './../api/Recording.api.js';
+
+export default {
+  name: "HomeGroupItem",
+  props: {
+    group: {
+      type: Object,
+      required: true
+    }
+  },
+  data: function () {
+    return {
+      count: 0
+    };
+  },
+  computed: {
+    ...mapState({
```

Comment:
Does groups actually get used on this component - isn't it about a single group?